backend/src/configuration/config.ts

```diff
@@ -75,7 +75,7 @@ const configuration = (): ConfigInterface => {
   return {
     APP: {
       PORT: port,
-      HOST: `http://${host}:${port}`,
+      HOST: host,
```

Comment:
It would be useful to have a real server host + port in the general app config (CONFIG.APP.HOST) in dev environment.